### DISCUSSION TITLE
How to get managed configurations for Apple applications?

### DISCUSSION THREAD

**Discussion by ashishmeher216** [micromdm/nanomdm#136] [Q&A]
Hey everyone, I cannot find managed configurations for Apple apps for all apps. In the case of Android, we get managed app configurations from Google itself. Is there any way I can get managed configuration keys-values for Apple apps? Do the MDM vendors contact the app developer for the same? I think that would not be feasible for all the app store apps.

https://developer.apple.com/documentation/devicemanagement/installenterpriseapplicationcommand/command/configuration

Please provide your feedback. Thanks

**Comment by korylprince** [comment]
Are you asking for iOS or macOS? You mention Android, so I assumed you meant iOS, but you linked to InstallEnterpriseApplication, which is only supported on macOS.

Either way, for Configuration Profiles and Managed App Configuration, it's up to the app vendor to document what keys they support. I'm not aware any other way to discover the keys an App supports.

I think most commercial MDM vendors provide a way for IT admins to set arbitrary Configuration Profile or App Configuration keys, so they don't need to know about every app.
